internal/dtos/discoveryServerConfigDto.go

```diff
@@ -0,0 +1,20 @@
+package dtos
+
+type DiscoveryClientConfigDto struct {
```

Comment:
You can remove the `Dto` suffix to your models.

Also this file should be moved to models folder.

Reply:
@kingster this is used to store the input parameter from discovery config file hence did not store in models. Aren't models related to persistence storage?

Reply:
My bad, Yes you are correct, the right place is the opts https://github.com/flipkart-incubator/dkv/blob/master/internal/opts/config.go which already hosts the the configs